Src/GhostTextBuffer.cpp

```diff
@@ -288,6 +290,21 @@ bool CGhostTextBuffer::InsertText (CCrystalTextView * pSource, int nLine,
 	else
 		bDiscrepancyInInsertedLines = false;
 
+	if (bSpecialLastLineHandling)
+	{
+		// The special case of inserting text into the very last line of a file when
+		//	that last line is marked as LF_GHOST.  Effectively, the new text is 
+		//	supposed to go "before" the Ghost, but mechanically the text is inserted
+		//	into the Ghost itself, with a new Ghost line appearing at the end of the
+		//	file.  Later (below), the Ghost status of both the first and last inserted 
+		//	lines will get straightened out, with the trailing Ghost line becomming  
+		//	a NULL line.
+		if ((GetLineFlags(nLine) & LF_GHOST) == 0)	// first line still marked GHOST
```

Comment:
@GreyMerlin This PR seems to cause the crash of issue #104.
I don't still understand about bSpecialLastLineHandling but if-condition and the comment in line 302 is contradict.
`if ((GetLineFlags(nLine) & LF_GHOST) != 0)`  seems to avoid the crash. Is this fix OK?